packages/frontend/src/components/ReactHookForm/CronInput/index.tsx

```diff
@@ -78,6 +86,7 @@ export const CronInternalInputs: FC<
                     error={error}
                 />
             )}
+            {children}
```

Comment:
Should we hide the timezone picker for hourly frequency?
If so, should we set timezone to `undefined`?

Reply:
I would keep the timezone picker.
Fun fact time... https://www.timeanddate.com/time/time-zones-interesting.html

Reply:
Oh that's good to know!!! 🙏